jdk-postgres/8-15/docker-entrypoint.sh

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env bash
```

Comment:
For now, it's fine, but I would like to explore if we can rewrite this with other languages than shell script if possible to make it more testable.
I understand shell script is the easiest way to do this kind of thing, but 300+ lines shell script is most probably not well-maintainable.

Reply:
I agree that this would not be easy to maintain. I intended to keep the `jdk8-postgres15` identical to the official `postgres` image, so this shell script originally comes from the official one [here](https://github.com/docker-library/postgres/blob/9b2559be2f13d24554516da5217950b2d41c447b/docker-entrypoint.sh). We can rewrite it in another language, but we have to be fully responsible for maintaining it afterward. What do you think?

Reply:
Ah, OK, I didn't know it is a fork of the official one. Then, as long as it is just a copy of the official one, I don't think we should rewrite it. Thanks!

BTW, is there any place we can note that this is a copy of the original one?

Reply:
The top of this file would be fine. I left that kind of comment in the `Dockerfile`, which is also forked from the original repo, but I didn't comment in this file for some reason. I'll add it in the subsequent PRs.